.cargo/config-release.toml

```diff
@@ -0,0 +1,11 @@
+# This is a config file used by Docker build process when building release version of drive-abci.
+# Hopefully it can be removed once profile-rustflags is stabilized, see https://github.com/rust-lang/cargo/issues/10271
+
+[target.aarch64-unknown-linux-musl]
```

Comment:
I think you can do something like that:

```
[target.aarch64-unknown-linux-musl]
rustflags = ["-C", "target-feature=-crt-static"]

[target.aarch64-unknown-linux-musl.profile.dev]
rustflags = ["-C", "target-feature=-crt-static", "--cfg", "tokio_unstable"]

[target.x86_64-unknown-linux-musl]
rustflags = ["-C", "target-feature=-crt-static"]

[target.x86_64-unknown-linux-musl.profile.dev]
rustflags = ["-C", "target-feature=-crt-static", "--cfg", "tokio_unstable"]

[target.aarch64-unknown-linux-gnu]
linker = "aarch64-linux-gnu-gcc"

[build.profile.dev]
rustflags = ["--cfg", "tokio_unstable"]
```

Reply:
Unfortunately, it doesn't work on `stable`.

Reply:
Bump Rust version? )))

Reply:
error: expected a string, but found a table for `dev`